### ISSUE TITLE
 Clouds visible through each other

### ISSUE THREAD

**Issue by dEffoZ** [ballisticfox/RSS-Reborn#14]
At different altitudes, clouds become visible through each other, or clouds that are higher than others overlap with those lower. How can we solve this problem?
![cc3](https://github.com/user-attachments/assets/6cfd08c9-6a65-4358-a2b4-0ae0544c1948)
![cc4](https://github.com/user-attachments/assets/60737bc5-88d9-426f-8964-53f5154a7dc4)
![cc5](https://github.com/user-attachments/assets/fbfa3c04-7012-4a68-854e-eb91cb793d14)
![cc6](https://github.com/user-attachments/assets/eaf7a035-be8c-4112-b34e-ccfd82c4b4b0)
![cc7](https://github.com/user-attachments/assets/fcde1bb8-c0b3-4cab-bfa9-1988d02dfe30)

**Comment by ballisticfox** [comment]
This is a known issue and needs to be fixed on EVE's end.

**Comment by dEffoZ** [comment]
So, now there is no way that I can fix it myself?

**Comment by ballisticfox** [comment]
Correct, it's due to the layers not having proper depth buffers, meaning they're rendered in an indiscriminate order instead of sorting the pixels from back to front.
